tools/tox_tools/dose_responses/dose_response.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0">
+<tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="0.1.0+galaxy0">
```

Comment:
Please keep `version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0"`

Otherwise this would be a downgrade.